spec/features/users_can_view_projects_spec.rb

```diff
@@ -1,18 +1,18 @@
 require "rails_helper"
 
 RSpec.feature "Users can view a list of projects" do
+  include_context "Academies API returns an establishment" do
+    let(:urn) { any_args }
```

Comment:
Are you mocking any value for urn becuase of the three projects? This is one of the reasons I lean more towards a helper, I like the explicit setup for the spec so it documents the state of the system under test, like we have `sign_in_with_user()` I see something like `mock_project_data_for(@unassigned_project.urn)` - what are your thoughts?